src/App.js

```diff
@@ -1,21 +1,29 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import Form from './components/Form'
 import ProductsList from './components/ProductsList'
 import Drawer from './components/Drawer'
```

Comment:
Was ist bei Dir der Drawer? Vielleicht eine Überlegung wert, ob jeder beim Lesen Deines Codes versteht, was Du damit meinst. Kann aber auch sein, dass nur ich hier hänge :)

Reply:
Der Drawer verhält sich tatsächlich wie eine Schublade: er fährt von unten ins Bild hinein und fährt wieder nach unten, um sich zu schließen. So wird er auch in verschiedenen UI libraries bezeichnet. 😊